graylog2-server/src/main/java/org/graylog2/rest/resources/cluster/ClusterMetricsResource.java

```diff
@@ -49,12 +49,14 @@
 import javax.ws.rs.core.HttpHeaders;
 import javax.ws.rs.core.MediaType;
 import java.io.IOException;
+import java.util.Map;
+import java.util.Optional;
 
 import static javax.ws.rs.core.Response.Status.BAD_GATEWAY;
 
 @RequiresAuthentication
 @Api(value = "Cluster/Metrics", description = "Cluster-wide Internal Graylog metrics")
-@Path("/cluster/{nodeId}/metrics")
+@Path("/cluster")
```

Comment:
Hmm, good catch. I guess the only way out here is to create another resource class.